services-worker/src/main/java/edu/unc/lib/dl/cdr/services/text/FullTextEnhancement.java

```diff
@@ -120,10 +122,9 @@ public Element call() throws EnhancementException {
 			}
 		} catch (FileSystemException e) {
 			throw new EnhancementException(e, Severity.FATAL);
-		} catch (NotFoundException e) {
-			throw new EnhancementException(e, Severity.UNRECOVERABLE);
-		} catch (FedoraException e) {
-			throw new EnhancementException("Full Text Enhancement failed to process " + dsid, e, Severity.RECOVERABLE);
+		} catch (ObjectExistsException | OptimisticLockException e) {
+			throw new EnhancementException("Failed to update object " + pid + " with " + dsid,
```

Comment:
maybe use string-replacement here, too, to match the way you wrote the other catch-blocks?

Reply:
EnhancementException doesn't have a string formatting method, that's mainly available through the logger